### PR TITLE
TF-859: Keep local copy in forwards

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/185774026-f823f399-739c-485b-b9e5-44f0cfd169eb.mov

https://user-images.githubusercontent.com/107173849/185774033-c12a5a48-d1b9-41a7-9364-76092e487212.mov

### CODE REVIEW

**Comment by hoangdat** [comment]
please let reviewer know what commits need to be review? @ManhNTX

**Comment by ManhNTX** [comment]
> please let reviewer know what commits need to be review? @ManhNTX
Screen Shot 2022-08-22 at 10 34 49
Check this

File: lib/features/manage_account/domain/usecases/edit_local_copy_in_forwarding_interactor.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+class EditLocalCopyInForwardingInteractor {
+  final ManageAccountRepository manageAccountRepository;
+
+  EditLocalCopyInForwardingInteractor(this.manageAccountRepository);
+
+  Stream<Either<Failure, Success>> execute(
+      AccountId accountId,
+      EditLocalCopyInForwardingRequest editRequest,
+      ) async* {
```
Comment (dab246) [inline lib/features/manage_account/domain/usecases/edit_local_copy_in_forwarding_interactor.dart:19]: remove indent

File: lib/features/manage_account/presentation/forward/forward_controller.dart [hunk 2]
```diff
@@ -269,7 +276,37 @@ class ForwardController extends BaseController {
...
+              keepLocalCopy: !currentForward.value!.localCopy)));
+    }
+  }
+
+  void _handleEditLocalCopySuccess(EditLocalCopyInForwardingSuccess success) {
+    if (currentOverlayContext != null && currentContext != null) {
+      _appToast.showToastWithIcon(
+        currentOverlayContext!,
+        message: success.forward.localCopy ?
+        AppLocalizations.of(currentContext!).toastMessageLocalCopyEnable :
```
Comment (dab246) [inline lib/features/manage_account/presentation/forward/forward_controller.dart:303]: indent

File: lib/features/manage_account/presentation/forward/forward_controller.dart [hunk 3]
```diff
@@ -269,7 +276,37 @@ class ForwardController extends BaseController {
...
+        AppLocalizations.of(currentContext!).toastMessageLocalCopyDisable,
```
Comment (dab246) [inline lib/features/manage_account/presentation/forward/forward_controller.dart:304]: same

**Comment by hoangdat** [comment]
LGTM

**Review by dab246 — APPROVED** [review]

**Pull request merged**